config/routes.js

```diff
@@ -1,117 +1,21 @@
-/* eslint key-spacing:0 */
+/* eslint-disable key-spacing */
```

Comment:
Yay! This gutting of the routes file really highlights the gain in the reduced code and noise/complexity in the app.